### PR TITLE
Patches to envs.ALE

### PR DESCRIPTION
**Pull request by toslunar**
- Change grayscale formula to `0.299 R' + 0.587 G' + 0.114 B'`, where the values are gamma compressed.
- Fix `frame_skip` argument.
- Add and fix assertions.

### CODE REVIEW

File: chainerrl/envs/ale.py [hunk 1]
```diff
@@ -116,10 +118,9 @@ def current_screen(self):
         rgb_img = np.maximum(self.ale.getScreenRGB(), self.last_raw_screen)
         # Make sure the last raw screen is used only once
         self.last_raw_screen = None
-        assert rgb_img.shape == (210, 160, 3)
```
Comment (muupan) [inline chainerrl/envs/ale.py]: Screen sizes can be different from (210, 160) when you use wrong rom files, so it's better to check them.

**Comment by muupan** [comment]
Good catch for the grayscale formula. It seems like deepmind and openai use your formula, and so should we.

File: chainerrl/envs/ale.py (hunk 1 above)
Comment (toslunar) [reply chainerrl/envs/ale.py]: This change removes a double-check.  This line checks rgb-format and L125--127 checks screen sizes.

Comment (muupan) [reply chainerrl/envs/ale.py]: I see.

**Pull request merged**